src/watch_test.ts

```diff
@@ -238,90 +211,28 @@ describe('Watch', () => {
                 receivedObjects.push(obj);
             },
             (err: any) => {
-                doneCalled = true;
                 doneErr.push(err);
+                doneResolve();
             },
         );
 
-        verify(fakeRequestor.webRequest(anything()));
-
-        const [opts] = capture(fakeRequestor.webRequest).last();
-        const reqOpts: request.OptionsWithUri = opts as request.OptionsWithUri;
+        stream.emit('error', errIn);
 
-        expect(reqOpts.uri).to.equal(`${server}${path}`);
-        expect(reqOpts.method).to.equal('GET');
-        expect(reqOpts.json).to.equal(true);
+        await donePromise;
 
         expect(receivedTypes).to.deep.equal([obj1.type]);
         expect(receivedObjects).to.deep.equal([obj1.object]);
 
-        expect(doneCalled).to.equal(true);
         expect(doneErr.length).to.equal(1);
         expect(doneErr[0]).to.deep.equal(errIn);
-    });
-
-    it.skip('should handle server side close correctly', async () => {
```

Comment:
Worried that we're removing this test too.

Reply:
my bad, I readded but it is now skipped due to the node-fetch bug I mentioned (at least I think). To properly close a stream, one needs to call `destroy` which other tests are already performing and it works

Reply:
I fixed the test